art/estimators/object_detection/PyTorchFasterRCNN.py

```diff
@@ -99,10 +101,16 @@ def __init__(
             preprocessing=preprocessing,
         )
 
-        assert clip_values[0] == 0, "This classifier requires un-normalized input images with clip_vales=(0, max_value)"
-        assert clip_values[1] > 0, "This classifier requires un-normalized input images with clip_vales=(0, max_value)"
-        assert preprocessing is None, "This estimator does not support `preprocessing`."
-        assert postprocessing_defences is None, "This estimator does not support `postprocessing_defences`."
+        if self.clip_values is not None:
+            if self.clip_values[0] != 0:
+                raise ValueError("This classifier requires un-normalized input images with clip_vales=(0, max_value).")
+            if self.clip_values[1] <= 0:
+                raise ValueError("This classifier requires un-normalized input images with clip_vales=(0, max_value).")
```

Comment:
```suggestion
        if self.clip_values[0] != 0:
            raise ValueError("This classifier requires un-normalized input images with clip_vales=(0, max_value).")
        if self.clip_values[1] <= 0:
            raise ValueError("This classifier requires un-normalized input images with clip_vales=(0, max_value).")
```

Reply:
I'm not sure this is the same, it is indeed a property, but `None` is an allowed value. In that case, trying to access one of its fields (`[0]` or `[1]`) will result in an error.

Reply:
Yes, you are right.